datafusion/core/src/physical_planner.rs

```diff
@@ -121,7 +121,7 @@ fn create_function_physical_name(
         false => "",
     };
 
-    let phys_name = format!("{}({}{})", fun, distinct_str, names.join(","));
+    let phys_name = format!("{}({}{})", fun, distinct_str, names.join(", "));
```

Comment:
Are the changes in this file the only in the PR?

Reply:
Several one, but are all like this